wallet_ui/canister/index.ts

```diff
@@ -170,16 +186,23 @@ export const Wallet = {
     ]);
   },
   async create_canister(p: {
-    controller?: Principal;
+    result: Principal[];
```

Comment:
is result the right name for this argument? I think I'd just go with `controllers`

Reply:
```suggestion
    controllers: Principal[];
```